### ISSUE TITLE
No type found for

### ISSUE THREAD

**Issue by jeffpyke** [stephenh/ts-proto#35]
Hi, I am seeing an error message when compiling this proto.
Proto def (baz.proto)
```proto
syntax = "proto3";

message Baz {
  oneof type {
    Foo_Bar foo = 1;
  }
}

message Foo_Bar {

}
```
Error
```
FAILED!No type found for .Foo_Bar
Error: No type found for .Foo_Bar
    at Object.fail (node_modules/ts-proto/build/utils.js:21:11)
    at toModuleAndType (node_modules/ts-proto/build/types.js:272:46)
    at messageToTypeName (node_modules/ts-proto/build/types.js:266:28)
    at basicTypeName (node_modules/ts-proto/build/types.js:82:20)
    at Object.toTypeName (node_modules/ts-proto/build/types.js:276:16)
    at generateInterfaceDeclaration (node_modules/ts-proto/build/main.js:220:102)
    at visit (node_modules/ts-proto/build/main.js:33:34)
    at visit (node_modules/ts-proto/build/main.js:253:9)
    at Object.generateFile (node_modules/ts-proto/build/main.js:32:5)
    at request.protoFile.map.file (node_modules/ts-proto/build/plugin.js:18:29)--ts_proto_out: protoc-gen-ts_proto: Plugin failed with status code 1.
```
ts-proto version 1.12.0
Thanks in advance for any advice. I think this has to do with ts-proto internally using underscores as separators for nesting.

**Comment by stephenh** [comment]
Thanks for the report; this one was easier, and you were right, I wasn't keeping "TS names that use underscores as 'separators'" (to flatten the types) sufficiently separate from "protobuf names that use periods as separators". Should be fixed in 1.13.0.

**Comment by jeffpyke** [comment]
Awesome, thank you for the quick fix! Appreciated :)
